proposals/lip-0066.md

```diff
@@ -143,11 +153,9 @@ The logic of the functions `derive_master_SK` and `derive_child_SK` are specifie
 
 #### Derivation Path for BLS Keys
 
-Similarly to the specifications of [EIP 2334](https://eips.ethereum.org/EIPS/eip-2334), we use the path `m/12381/134/chainID/0` for the BLS key derived for the chain with ID `chainID`.
-
-If `chainID` is not known, use the path `m/12381/134/0/n` for the n-th generator key derived for chains without known chain ID.
+Similarly to the specifications of [EIP 2334](https://eips.ethereum.org/EIPS/eip-2334), we use the path `m/12381/134/chainID/0` for the BLS key derived for the chain with ID `chainID`. Here `chainID` is the big-endian decoding of `bytesToInt(chainID)`.
```

Comment:
same as above

Reply:
Changed the variable name 'chainID' to 'chainIDConverted'